website/source/docs/agent/services.html.md

```diff
@@ -111,19 +135,19 @@ from `1`.
 
 -> **Note:** There is more information about [checks here](/docs/agent/checks.html).
 
-The `enable_tag_override` can optionally be specified to disable the
-anti-entropy feature for this service. If `enable_tag_override` is set to
-`TRUE` then external agents can update this service in the
-[catalog](/api/catalog.html) and modify the tags. Subsequent
-local sync operations by this agent will ignore the updated tags. For
-example, if an external agent modified both the tags and the port for
-this service and `enable_tag_override` was set to `TRUE` then after the next
-sync cycle the service's port would revert to the original value but the
-tags would maintain the updated value. As a counter example: If an
-external agent modified both the tags and port for this service and
-`enable_tag_override` was set to `FALSE` then after the next sync cycle the
-service's port *and* the tags would revert to the original value and all
-modifications would be lost.
+### Enable Tag Override
+
+Services may also contain a `token` field to provide an ACL token. This token is
```

Comment:
This line felt like it belonged outside of the "Enable Tag Override" header, but then I kept reading and see why it ended up here. Maybe the header should be "Enable Tag Override & Anti-Entropy", or something?